tests/test_frompyobject.rs

```diff
@@ -6,6 +6,21 @@ use pyo3::PyMappingProtocol;
 #[macro_use]
 mod common;
 
+/// Helper function that concatenates the error message from
+/// each error in the traceback into a single string that can
+/// be tested.
+fn extract_traceback(mut error: PyErr) -> String {
```

Comment:
As this function seems to be called from tests which all run with the GIL, it'd be faster to just take `py: Python` as an argument rather than calling `acquire_gil`.

Reply:
:+1: